integration/templates/astro-node/src/pages/index.astro

```diff
@@ -1,10 +1,10 @@
 ---
 import Layout from "../layouts/Layout.astro";
 import Card from "../components/Card.astro";
-import SignOutButton from "../lib/astro-components/SignOutButton.astro";
```

Comment:
Instead of doing this custom component, I used the built-in button from `/components/unstyled` so it can be tested as well